modules/custom/lua/claim_shield.lua

```diff
@@ -158,16 +158,16 @@ end
 
 -- Main entrypoint of each override
 local overrideFunc = function(mob)
-    spawnFunc(mob)
+    mob:addListener('SPAWN', mob:getPacketName() .. '_CS_SPAWN', listenerFunc)
```

Comment:
Please use `string.format` or `fmt` for building strings, `..` is harder to follow